terraform/azure-pipelines.yml

```diff
@@ -26,8 +26,14 @@ stages:
             value: $[variables['Build.SourceBranchName']]
           - name: TARGET
             value: $[variables['System.PullRequest.TargetBranch']]
+          - name: IS_TAG
+            value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/')]
         steps:
-          - bash: python terraform/pipeline/workspace.py
+          - bash: |
+              python terraform/pipeline/workspace.py
```

Comment:
I think it's probably because this line didn't change, in that the `workspace` is still set as an output variable by `workspace.py`: https://github.com/cal-itp/benefits/pull/2213/files/f4304d09dd1bf08a2797f165ba7d603cd0743a48#diff-33a8fda4d235c86352a2629618f4d0829b626df6084edef709ad22d97e8f80fcR52